<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFoodWindow</class>
 <widget class="QWidget" name="AddFoodWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>航班餐食管理</string>
  </property>
  <widget class="QLabel" name="labelFlight">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入航班号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFlightID">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFood">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入菜品：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAddFood">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addFood">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteFood">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QTableView" name="foodList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
